tests/selenium/pbtest_org_test.py

```diff
@@ -7,112 +7,63 @@
 from selenium.common.exceptions import (NoSuchElementException,
                                         StaleElementReferenceException)
 
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
 # where to run the acceptance tests
 PBTEST_ORG_URL = "https://pbtest.org/tracker"
 
 # the id of the element where test results are reported
 PBTEST_ORG_TEST_RESULTS_TABLE_ID = "results"
 
 # unicode characters we look in the results to tell if a test passed or failed
-PASS = u'\u2713'
-FAIL = u'\u2717'
+PASS = u'Pass'
+FAIL = u'Fail'
 
 
 class PBTest_Org_test(pbtest.PBSeleniumTest):
     """Run the pbtest.org website acceptance tests. Loads the pbtest.org test
-       suite and assert that none of the tests failed. Ignores tests with a
-       status of "undefined" and any tests not visible to the user."""
+       suite and assert that none of the tests failed or are 'undefined'."""
 
+    @pbtest.repeat_if_failed(5)
```

Comment:
So now instead of the whole test suite getting re-run for this flaky test, only tests with this decorator get re-run. Which should greatly improve the speed of the travis runs.